LightBlue.MultiHost/Runners/ProcessExtensions.cs

```diff
@@ -13,5 +14,25 @@ public static IEnumerable<Process> GetChildren(this Process process)
             foreach (var mo in new ManagementObjectSearcher(query).Get())
                 yield return Process.GetProcessById(Convert.ToInt32(mo["ProcessID"]));
         }
+
+        public static void AllocateToMultiHostProcess(this Process process)
+        {
+            // Allows windows to clean up child process for us when the parent process is killed
+            MultiHostProcess.Job.AddProcess(process);
+        }
+
+        public static void SetPriority(this Process process, Role role)
+        {
+            if (role.Title.EndsWith("Hub"))
```

Comment:
Let's drive this from an explicit role configuration, rather than role title naming convention